src/native/fs/fs_napi.cpp

```diff
@@ -149,6 +159,22 @@ struct FSWorker : public Napi::AsyncWorker
             std::string errmsg = strerror(errno);
             SetError(errmsg);
         }
+        bool change_uid = orig_uid != _req_uid;
```

Comment:
SetSyscallError can be called multiple times in a single run.
In this case, you will set to orig_uid and orig_gid multiple times.

Reply:
will be removed